app/models/preprint.ts

```diff
@@ -56,6 +56,7 @@ export default class PreprintModel extends OsfModel {
     @attr('array') preregLinks!: string[];
     @attr('string') whyNoData!: string;
     @attr('string') whyNoPrereg!: string;
+    @attr('string') preregLinkInfo!: string;
```

Comment:
I think it'll be better to use an enum here, since there are only three valid options